io.github.trevorsandy.LPub3D.json

```diff
@@ -0,0 +1,274 @@
+{
+  "id": "io.github.trevorsandy.LPub3D",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.15",
+  "sdk": "org.kde.Sdk",
+  "command": "lpub3d24",
+  "rename-icon": "lpub3d",
+  "rename-desktop-file": "lpub3d.desktop",
+  "rename-appdata-file": "io.github.trevorsandy.appdata.xml",
+  "build-options": {
+    "env": {
+      "LP3D_DIST_DIR_PATH": "/app/lpub3d_linux_3rdparty",
+      "LDRAWDIR": "/app/lpub3d_linux_3rdparty/ldraw"
+    }
+  },
+  "cleanup": [
+    "/app/lpub3d_linux_3rdparty"
+  ],
+  "finish-args": [
+    "--device=dri",
+    "--share=ipc",
+    "--socket=x11",
+    "--filesystem=home",
+    "--env=TMPDIR=/var/tmp",
+    "--env=QT_ENABLE_HIGHDPI_SCALING=1"
+  ],
+  "modules": [
+    "shared-modules/glu/glu-9.json",
+    {
+      "name": "freeglut",
+      "build-options": {
+        "cflags": "-fcommon"
+      },
+      "buildsystem": "cmake-ninja",
+      "cleanup": [
+        "/include",
+        "*.a"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://downloads.sourceforge.net/project/freeglut/freeglut/3.2.1/freeglut-3.2.1.tar.gz",
+          "sha256": "d4000e02102acaf259998c870e25214739d1f16f67f99cb35e4f46841399da68"
+        }
+      ]
+    },
+    {
+      "name": "mesa",
```

Comment:
Why does it need its own mesa?

Reply:
@hfiguiere stated:

> And then there is the problem of 'osmesa' that is not in the runtime.

So I understood OSMesa is not available in the runtime Mesa installation.

Cheers,

Reply:
I saw. I can have a look like that later.